src/pages/AddLiquidityV2/components/DisclaimerERC20.tsx

```diff
@@ -1,13 +1,38 @@
 import { Trans } from '@lingui/macro'
 import { AlertTriangle } from 'react-feather'
 import { Flex, Text } from 'rebass'
+import { useGetTokenListQuery } from 'services/ksSetting'
 
 import { WarningCard } from 'components/Card'
+import { useActiveWeb3React } from 'hooks'
 import useTheme from 'hooks/useTheme'
 import { ExternalLink } from 'theme'
 
-export default function DisclaimerERC20({ href }: { href?: string }) {
+export default function DisclaimerERC20({ href, token0, token1 }: { href?: string; token0: string; token1: string }) {
   const theme = useTheme()
+  const { chainId } = useActiveWeb3React()
+  const { data: data0 } = useGetTokenListQuery(
```

Comment:
use this api instead: `addresses`

 https://ks-setting.kyberswap.com/api/v1/tokens?addresses=0x8d6cebd76f18e1558d4db88138e2defb3909fad6,0x111111111117dC0aa78b770fA6A738034120C302&chainIds=1

Reply:
Done